modules/tinymce/changelog.txt

```diff
@@ -1,5 +1,6 @@
 Version 5.4.0 (TBD)
     Changed `mceInsertTable` command and `insertTable` API method to take optional header rows and columns arguments #TINY-6012
+    Fixed the code that filters commands out of the `undo` stack. It is now case insensitive.
```

Comment:
Not a hard rule, but we usually use one-sentence changelog messages. I would write this as `Fixed filters for removing commands from the undo stack to be case-insensitive.` or something I think. Not sure if we care enough to change it though. Anyone else particularly care either way?

Reply:
Noted and changed. Thanks!